### PR TITLE
[Documentation] `bundle add` documentation

### PR DESCRIPTION
**Pull request by denniss**

### CODE REVIEW

File: man/bundle-add.ronn [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## SYNOPSIS
+
+`bundle add` [--group=GROUP] [--version=VERSION] [--source=SOURCE]  [GEM]
+
+## DESCRIPTION
+Adds the named gem to Gemfile and run `bundle install`.
+
+Example:
+
```
Comment (colby-swandale) [inline man/bundle-add.ronn]: There needs be an example without any options,  ie: `bundle add rails` (this should be the first example)

File: man/bundle-add.ronn [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+bundle add "rails" --version "< 3.0, > 1.1"
+
+bundle add "rails" --version "~> 5.0.0" --source "https://gems.example.com" --group "development"
```
Comment (colby-swandale) [inline man/bundle-add.ronn]: are the quotes required for a gem name like "rails"?

File: man/bundle-add.ronn [hunk 3]
```diff
@@ -0,0 +1,27 @@
+bundle-add(1) -- Add gem to Gemfile and run bundle install
+================================================================
+
+## SYNOPSIS
+
+`bundle add` [--group=GROUP] [--version=VERSION] [--source=SOURCE]  [GEM]
+
+## DESCRIPTION
+Adds the named gem to Gemfile and run `bundle install`.
```
Comment (colby-swandale) [inline man/bundle-add.ronn]: `named gem to the Gemfile`

File: man/bundle-add.ronn [hunk 4]
```diff
@@ -0,0 +1,27 @@
+bundle-add(1) -- Add gem to Gemfile and run bundle install
+================================================================
+
+## SYNOPSIS
+
+`bundle add` [--group=GROUP] [--version=VERSION] [--source=SOURCE]  [GEM]
```
Comment (colby-swandale) [inline man/bundle-add.ronn]: The gem name is required for `bundle add` so this is incorrect.
```
bundle add <GEM_NAME> [--group=GROUP] [--version=VERSION] [--source=SOURCE]
```

File: man/bundle-add.ronn [hunk 5]
```diff
@@ -0,0 +1,27 @@
+bundle-add(1) -- Add gem to Gemfile and run bundle install
```
Comment (colby-swandale) [inline man/bundle-add.ronn]: `Add gem to the Gemfile`

**Comment by denniss** [comment]
@colby-swandale Fixed the requested changes! Thanks

**Comment by colby-swandale** [comment]
thank you!

@bundlerbot r+

**Pull request merged**